cypress/integration/viewAllApis.spec.js

```diff
@@ -1,10 +1,5 @@
 import { screenSizes } from "../support/screenSizes";
 
-describe("API Catalogue Page is limited to signed in users", () => {
```

Comment:
Why are you removing this test?

Reply:
It has been replaced with the test in [cypress/integration/routing.spec.js](https://github.com/LBHackney-IT/developers-hub-frontend/blob/e8675633c15bfeefcc11eeead59c08d8e1ddc8bf/cypress/integration/routing.spec.js)!

This is because authentication logic is no longer tied to individual pages, but to the routing as a whole